### DISCUSSION TITLE
Planner for external networks

### DISCUSSION THREAD

**Discussion by JBirSP** [Azure/ipam#338] [Q&A] [answered]
Hi!

I find the planner and generator features very useful for our Azure environments. I would like to use those tools (especially the planner) for our office networks as well. Is this possible?

**Comment by DCMattyG** [comment]
Hi @JBirSP, of course you can.

By creating a Space/Block that represents your office network, you can user the **Externals** feature to create a CIDR that is outside of Azure. When you create an **External** network, this will mark that CIDR space as used for the given Block and will reflect as such when using tools like the **Planner**

Furthermore, you can define Subnets & Endpoints for on-premises networks if you so choose within the **Externals** configuration section as well.

I hope that helps, please let me know if you need any further clarification on this feature!

**Reply by DCMattyG** [reply, marked as answer]
Actually, I take that back @JBirSP...only Azure Virtual Networks show up currently using the Planner tool, not External Networks.

I'll get this addressed right away. I agree this should be available for both Azure and External Networks within the tool.

This will be available in Azure IPAM v3.6.0.

**Reply by JBirSP** [reply]
Thanks! We'll be looking forward to that update 👍
